corehq/apps/app_manager/tests/test_bulk_app_translation.py

```diff
@@ -219,8 +219,10 @@ def test_change_upload(self):
         )
 
         # Test special characters and output refs
-        self.assert_question_label("one &lt; two", 0, 0, "en", "/data/question2")
-        self.assert_question_label("un &lt; deux", 0, 0, "fra", "/data/question2")
+        #jls
+        self.assert_question_label("one < two", 0, 0, "en", "/data/question2")
+        self.assert_question_label("un < deux", 0, 0, "fra", "/data/question2")
+        self.assert_question_label("question3's label", 0, 0, "fra", "/data/question3")
```

Comment:
Possibly add a test for something like `blah < blah <output value='blah' /> <blah />` ?